app/controllers/profiles_controller.rb

```diff
@@ -0,0 +1,42 @@
+class ProfilesController < ApplicationController
+  def show
+    @profile = Profile.find(params[:id])
+  end
+
+  def new
+    return redirect_to root_path if current_client.enroll.nil?
+
+    @profile = Profile.new
+  end
+
+  def create
+    @enroll = Enroll.find(current_client.id)
+    @profile = Profile.new(params_profile)
+    if @profile.save
+      redirect_to @profile, notice: t('.successfully')
+    else
+      render :new
+    end
+  end
+
+  def edit
+    @profile = Profile.find(params[:id])
+  end
+
+  def update
+    @enroll = Enroll.find(current_client.id)
+    @profile = Profile.find(params[:id])
+    @profile.update(params_profile)
```

Comment:
```suggestion
    @profile.update(**params_profile, enroll: @enroll)
```

Pra não ter que dar merge lá embaixo e ficar bem confuso daria pra fazer uso do double splat operator aqui, vou deixar um exemplo. Acho que daria certo mas não testei.

Reply:
Ou melhor ainda se é um update poderia talvez fazer @enroll.profile.update porque aí nem precisa do enroll já que tá anexado já

Reply:
Usamos essa sugestão  @profile.update(**params_profile, enroll: @enroll) realmente fica mais claro.  A outra que voce sugeriu acabou gerando outros problemas com o "def new"   hehe
Adicionamos mais um atributo ao profile e somente isso. 
Decidimos ter um perfil de aluno com os seus dados apenas apos ter feito a matricula, ficou mais como um extra mesmo mas não afeta em nada o fluxo apos o aluno ter ser matriculado. Talvez no futuro possamos permitir que o personal possar ver o perfil dos seus alunos